normalizer/app/geo_processor.py

```
@@ -35,131 +39,190 @@ def replacer(match):
+    if dist >= distance_threshold:
+        if logger:
+            pre_msg = f"警告: {index}行目 '{address}' のGoogle座標と国土地理院座標が {int(dist)}m ズレ。"
+            if priority == "gsi":
+                logger(pre_msg + "国土地理院APIの座標を採用します。")
+            elif priority == "google":
+                logger(pre_msg + "Google座標を採用します。")
+        return (lat2, lon2, "gsi") if priority == "gsi" else (lat1, lon1, "google")
+    # ズレが閾値未満ならGoogle優先
+    return lat1, lon1, "google"
+
+def render_template(index, template_str, row, cache, full_api_address, api_key, sleep_msec, gsi_check, gsi_dist, priority, logger=None):
```

Pylint (3.3.7)

[refactor] Too many arguments (11/5)

(R0913)